package/requirements.txt

```
@@ -1,5 +1,6 @@
 semver~=2.10 # Needs to be at least 2.10.0 to get VersionInfo.match
 kedro>=0.16.0
+kedro-telemetry>=0.1.0
```

I would've been a little more restrictive but up to you.
```suggestion
kedro-telemetry~=0.1.0
```

---

Yea my passive-agressive way of saying if you break my software, I will come find you :D